time stamp
string frame_id
string robot_location
string person_location
